discovery_climate_music_analysis/getters/collect_songs.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
```

Comment:
Could you please also add a general docstring at the top of the file, with a couple sentences about what these utils do? Something like

```
"""
Getters for fetching songs from Spotify API...
"""